helpers/routeHelpers.js

```diff
@@ -40,7 +40,21 @@ export const schema = {
             name: Joi.string()
         })
     },
+    role: {
+        post: Joi.object().keys({
+            name: Joi.string().required(),
+            color: Joi.string().required()

```

Comment:
There is no validation for colors. It should be hexadecimal colour or just enum with specific colors